services/web/server/src/simcore_service_webserver/security_roles.py

```diff
@@ -78,6 +78,7 @@
     UserRole.TESTER: {
         "can": [
             "diagnostics.read",
+            "storage.files.sync",
```

Comment:
For later: shouldn't we create a higher user role like admin for these operations? It is an operation assigned to an framework administrator more than a tester

Reply:
yes. that would make sense.